db/dcrpg/upgrades.go

```diff
@@ -486,6 +516,48 @@ func (pgb *ChainDB) handleVinsTableMainchainupgrade() (int64, error) {
 	return rowsUpdated, nil
 }
 
+func (pgb *ChainDB) handleTicketsBlockTimeUpgrade() (int64, error) {
+	log.Info(" - Retrieving all the ticket transactions with blocktime")
```

Comment:
` - Retrieving block times for all ticket transactions.`

Reply:
ok. Noted